### PR TITLE
add-member-profile-of-philip

### LINKED ISSUE #4440: [Community] Member Profile: Philip
**Current Behavior**

@philip-21 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @philip-21 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @philip-21
- Twitter: 
- LinkedIn: [philip-obiora-aa09401a9](https://www.linkedin.com/in/philip-obiora-aa09401a9/)
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR fixes #4440 
- creates @philip-21's member profile on https://layer5.io/community/members.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 93f3994fdf3fb59d4a2a6665b56380c9666fd514 at: https://6499aafdf5da58035d22505e--layer5.netlify.app

**Comment by leecalcote** [comment]
@vishalvivekm be sure to assign Philip for review.

**Comment by l5io** [comment]
🚀 Preview for commit 93f3994fdf3fb59d4a2a6665b56380c9666fd514 at: https://649b13f27e004b114a486de7--layer5.netlify.app

**Review by samyakshah3008 — APPROVED** [review]
Looks good to merge from my side ✅ 
We can wait for Philip to review his own member's profile before merging.

**Review by DarrenDsouza7273 — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
@Philip-21?

File: src/collections/members/philip-obiora/index.mdx
Comment (leecalcote) [inline src/collections/members/philip-obiora/index.mdx:12]: Let's add the `meshery` badge to Philip's profile ✅

**Comment by Philip-21** [comment]
Everything is perfect everyone thanks 😊

File: src/collections/members/philip-obiora/index.mdx
Comment (Philip-21) [reply src/collections/members/philip-obiora/index.mdx:12]: Thanks I just accepted the badge 🙂

**Review by goldfishdolphin — APPROVED** [review]
Looking good 👍🏽

File: src/collections/members/philip-obiora/index.mdx
Comment (leecalcote) [reply src/collections/members/philip-obiora/index.mdx:9]: Oh, good catch. While we're at it, let's lowercase "Open source" and "Cloud", too.

@goldfishdolphin, perhaps, you'd like to make these changes directly. Press CTL+G (or CMD+G) while inside of a comment and you should be able to suggest these changes.

**Comment by l5io** [comment]
🚀 Preview for commit b5b6ee04d596a7bdf0ed9a73fd17bbe816f66d13 at: https://649ea84739c2255b1434a02f--layer5.netlify.app

**Review by goldfishdolphin — APPROVED** [review]
Looking good

File: src/collections/members/philip-obiora/index.mdx
Comment (goldfishdolphin) [reply src/collections/members/philip-obiora/index.mdx]: That’s a great idea 💡

**Comment by leecalcote** [comment]
Whee! 🎈

**Pull request merged**
